### ISSUE TITLE
Add crafting of fluids

### ISSUE THREAD

**Issue by chaos234** [ExtraCells/ExtraCells2#276]
Hey,

I tried out the combination of ME Fluid Interface with 4-6 ME Fluid Assembler and included some patterns to create the fluid which I want but the Interface(s) are not listed in the Interface Terminal.

Also it doesn't seems that the ME Fluid Assembler crafts the wanted fluid like the combination of e. g. redstone to destab. restone to destab. redstone bukkit in the magma crucibal and fluid transposer.

So why is this not working? I've the interface and the requ. assembler (like the Molecular Assembler for AE2 patterns) and the same correct setup like the Interface + Molecular Assembler.

Please fix this otherwise the fluid system (ME Fluid Interface, ME Fluid Assembler) is bullshit - sorry but this is my opinion!

**Comment by Buzzefall** [comment]
It works the other way, the assembler holds patterns in itself and there is no interface through terminal. 
To use fluid in order to craft a thing place buckit with fluid in crafting grid in pattern terminal.

**Comment by chaos234** [comment]
This won't work. If I want gelid cryotheum from the dust then over the interface/assembler without any buckit. Also I testet it with the buckit and with the liquid icon but nothing worked.
It should be like the AE2 interface and assembler which can store upto 9 fluid/liquid/molten patterns which are created over the empty pattern and after that combined with the raw materiel type, e. g. Redstone for Destabelized Redstone or cryo-dust for gelid cryo and so on. Then the craftingsystem sends the rawmaterials to the assembler and it will "morph" it into the liquid. Should be coust a defined amount of energy for this process like the ones for the inscriber.
This is, what the fluid system should do and not the actually functionality. Acutally I can't craft the fluids like the above one, so I must create them first over magma crucibal and this isn't acceptable. Because I also don't need a crafting table to do shaped/shapless crafting with AE2 because there for are the interfaces in combination with the assembler and this should be used for liquids to!

Von Samsung-Tablet gesendet

-------- Ursprüngliche Nachricht --------
Von: Buzzefall notifications@github.com 
Datum: 18.08.2015  08:53  (GMT+01:00) 
An: ExtraCells/ExtraCells2 ExtraCells2@noreply.github.com 
Cc: Chaos234 info@deltanet-production.de 
Betreff: Re: [ExtraCells2] Fluid Crafting not working (#276) 

It works the other way, the assembler holds patterns in itself and there is no interface through terminal.

**Comment by Buzzefall** [comment]
I automated it with just using crucible. It just doesn't count any bucket
because there are none produced, only liquid directly into ME. Whatever.
Assembler takes liquids directly from ME when you place buckit in recipe. For me that's enough, ppl make mod on their free time and thanks to them at all.

2015-08-18 19:40 GMT+06:00 Chaos234 notifications@github.com:

**Comment by chaos234** [comment]
And THIS isn't acceptable! There are posibillities for autocrafting with interface and assembler which don't need any additional components like the crucible otherwise we can delete those components (assembler/interface) because we don't need them because they don't do that what they should have to do ... crafting fluids with the rawtypes without any other requierements.

Von Samsung-Tablet gesendet

-------- Ursprüngliche Nachricht --------
Von: Buzzefall notifications@github.com 
Datum: 18.08.2015  16:02  (GMT+01:00) 
An: ExtraCells/ExtraCells2 ExtraCells2@noreply.github.com 
Cc: Chaos234 info@deltanet-production.de 
Betreff: Re: [ExtraCells2] Fluid Crafting not working (#276) 

I automated it with just using crucible. It just doesn't count any bucket

because there are none produced, only liquid directly into ME. Whatever.

2015-08-18 19:40 GMT+06:00 Chaos234 notifications@github.com:

**Comment by ruifung** [comment]
Tone down the aggressiveness, just reading your reply makes me feel like you feel you are entitled to something here. 

If I'm understanding it clearly, you want to be able to request liquid to craft?
If that's the case, it's probably a limit of the AE2 Auto crafting, as far as I can tell  AE2's autocrafting is designed to operate exclusively with items, and I don't think anyone is willing to re-implement a entire autocrafting system just for fluids.

**Comment by chaos234** [comment]
It's simple.
You yust make the pattern you want and put it in the interface and then you start the crafting. Now the interface puts the rawtype e.g. redstone in the assembler and with AE-Power it will be morphed in the liquid like you can copy Items over a pattern to create them with UU-Mater but at this point you do that with the rawtype from the liquid.

Von Samsung-Tablet gesendet

-------- Ursprüngliche Nachricht --------
Von: ruifung notifications@github.com 
Datum: 18.08.2015  17:05  (GMT+01:00) 
An: ExtraCells/ExtraCells2 ExtraCells2@noreply.github.com 
Cc: Chaos234 info@deltanet-production.de 
Betreff: Re: [ExtraCells2] Fluid Crafting not working (#276) 

Tone down the aggressiveness, just reading your reply makes me feel like you feel you are entitled to something here. 

Also I can't understand what you want clearly, "crafting fluids with the rawtypes without any other requirements" I do not understand what you mean by that.

**Comment by ruifung** [comment]
So... you are saying you want to create e.g. redstone from molten redstone?
(If its that then no, there isn't really a central mapping of item to fluids I think?)

And I still can't understand what you are trying to say clearly.

**Comment by chaos234** [comment]
Falsmdirection but for that you can create a special pattern but more to that later.
Well, you want to have destabilized redstone then you create the pattern for that. But because the icon of destabilized redstone is not declared with an amount you have no limits. So you can define one or you say "ok, 1 redstone are 100mB destab redstone" then it puts the redstone in the assembler and with e.g. 20kAE it morphes it into destab.redstone or in that possible liquid whuch is available pending on the created pattern.
At the other side, if you have the liquid, then you can say "well, creating liquids is so easy, time to make the other side a little bit harder". Now you design some templates e.g. for redstone or block of redstone with the needed amount for the liquid which can be pasted into the assembler to tell them tha you are creating non-liquids. Or you use the item as template and give them an internal damge value how often it can be used as template (10 times? 100 times? 1000 times? ...), if it can be used to get any liquid from that item.
This should work perfectly ;).

Von Samsung-Tablet gesendet

-------- Ursprüngliche Nachricht --------
Von: ruifung notifications@github.com 
Datum: 18.08.2015  17:35  (GMT+01:00) 
An: ExtraCells/ExtraCells2 ExtraCells2@noreply.github.com 
Cc: Chaos234 info@deltanet-production.de 
Betreff: Re: [ExtraCells2] Fluid Crafting not working (#276) 

So... you are saying you want to create e.g. redstone from molten redstone?

**Comment by ruifung** [comment]
And do tell, where is there a central location where all these recipes are stored? This would need to add integration for like.. every other mod that adds their own machines to make their own liquid to handle that.

And letting the player define the input item and output liquid without any way to verify?

**Comment by XFactHD** [comment]
@Chaos234 The "morphing" you are describing doesn't make any sense to me. If this was the case you would also have to be able to just put for example copper dust into a molecular assembler, use up ginormous amounts of energy and then get copper ingots back.
I think that the use of the ME Fluid Assembler is to create blocks out of fluids (obsidian out of water and lava) or create another fluid out of two or more fluids. Just look at the MFR LiquiCrafter, it seems to be something similar.

**Comment by ruifung** [comment]
same function, different execution.

**Comment by ruifung** [comment]
duplicate of #241 (at least if I understand what you want correctly)
